### ISSUE TITLE
box, object, and class loss

### ISSUE THREAD

**Issue by larrywal-express** [ultralytics/yolov5#5199]
## ❔Question
Which one of the following: box loss, object loss, and class loss must overfit before ending training or determines accuracy?

**Comment by glenn-jocher** [comment]
@larrywal-express up to you. It's desirable to have all your loss components overfit similarly of course, but this is hard to achieve in practice.

**Comment by larrywal-express** [comment]
@glenn-jocher that means all must overfit even if for example my object loss is overfitting while the loss of box and class kept on decreasing ？

**Comment by glenn-jocher** [comment]
@larrywal-express your losses overfitting is synonymous with your validation mAP decreasing, so if you want to look at single metric rather than multiple losses you can also go that route. The basic concept is just to have a sense of having trained sufficiently.

**Comment by larrywal-express** [comment]
@glenn-jocher Thanks for your time.
